doc/sphinx/practical-tools/coqide.rst

```diff
@@ -369,16 +399,16 @@ and set the third item in the list as desired, such as in the third line.
 Avoid assigning the same binding to multiple items.
 
 If the same menu item name appears on multiple lines in the file, the value from the
-last line is used.  This is convenient for copying a group of changes from elsewhere--just
-insert the changes at the end of the file.  The next time CoqIDE terminates, it will
+last line is used.  This is convenient for copying a group of changes from elsewhere –
+just insert the changes at the end of the file.  The next time CoqIDE terminates, it will
```

Comment:
```suggestion
last line is used.  This is convenient for copying a group of changes from elsewhere–just
insert the changes at the end of the file.  The next time CoqIDE terminates, it will
```

The doc follows American usage (not my choice BTW), which doesn't use spaces before and after an en dash.  While I couldn't find an internet reference, you can see this usage in the Chicago Manual of Style.

Reply:
Strange American usage. OK, restored.